datagateway_api/src/search_api/filters.py

```diff
@@ -162,6 +165,14 @@ def apply_filter(self, query):
         return super().apply_filter(query.icat_query.query)
 
 
+class SearchAPIQueryFilter(PythonICATQueryFilter):
```

Comment:
As per my previous comment, you can inherit directly from `ScoringQueryFilter`. Would you be able to rename this class to `SearchAPIScoringFilter` please? When searching for `SearchAPIQueryFilter`, it also returns results for `SearchAPIQueryFilterFactory`, unless you have a match whole word filter set.